src/components/hero/DemoButton.tsx

```diff
@@ -34,7 +32,9 @@ function DemoButton({ step, type }: Props) {
 
     const objectRoles = useEntitiesStore_capabilities_adminable();
 
-    const demoAccessExists = useEntitiesStore_hasDemoTenantAccess();
+    const demoAccessExists = useUserInfoSummaryStore(
+        (state) => state.hasDemoAccess
+    );
```

Comment:
Another repeated change. `demoAccessExists` should now fetch from the user info summary store.